wiki/markdown/running-eiffel-intelligence-frontend.md

```diff
@@ -37,8 +46,43 @@ remember to keep the name of the properties file if you are a beginner to
 Spring. More advanced Spring user can look [here](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html)
 for more information about using external configuration.
 
-If only few properties need to be overriden then use Java opts, for example
+If only few properties need to be overriden, then use Java opts, for example
 
     java -jar eiffel-intelligence-frontend-<version>.war -Dspring.data.mongodb.port=27019
 
+
+## Running in Tomcat instance
+
+To run Eiffel-Intelligence front-end in Tomcat, the war file must be put into the webapp folder in tomcat installation folder, also called catalina home folder:
+
+    (catalina home)/webapp/
+
+If Eiffel-Intelligence front-end should be run without any conext-path in the url address, then overwrite ROOT.war file in webapp folder with eiffel-intelligence-frontend-<version>.war file:
+
+    cp eiffel-intelligence-frontend-<version>.war (catalina home)/webapp/ROOT.war
+
+Remove "ROOT" folder in webapp folder:
+
+    rm -rf (catalina home)/webapp/ROOT/
+
+Create "config" folder in webapp folder, if it not exists. Spring and Eiffel-Intelligence front-end will look for the application.properties configuration file in this config folder:
+
+    mkdir (catalina home)/webapp/config
+
+Copy the application.propeties file into the newly created config folder:
+    
+    cp application.properties (catalina home)/webapp/config
+
+Start Tomcat and Eiffel-Intelligence front-end in background/daemon mode by exectuing command:
+
+    (catalina home)/bin/catalina.sh start
+
+To run Tomcat and Eiffel-Intelligence front-end with logs printed to console:
+    
+    (catalina home)/bin/catalina.sh run
+
+## Eiffel-Intelligence fron-tend configurations and properties
+
 All available Eiffel-Intelligence front-end properties can be found in [application.properties](https://github.com/Ericsson/eiffel-intelligence-frontend/blob/master/src/main/resources/application.properties) example file.
+
+More documentation of each Eiffel-Intelligence front-end properties and configurations can be found in [Configuration page](./configuration.md)
```

Comment:
property.
In this case each should be followed by singular noun.

Reply:
fixed.